ACCESSIBILITY.md

```diff
@@ -0,0 +1,236 @@
+```diff
+-<div class="o-forms-field">
++<div class="o-forms-field" role="group" aria-labelledby="example-group-title">
+	<span class="o-forms-title">
+-		<span class="o-forms-title__main">Group Title</span>
++		<span class="o-forms-title__main" id="example-group-title">Group Title</span>
+		<span class="o-forms-title__prompt">Group info</span>
+	</span>
+	...
+</div>
+```
+
+Using `aria-labelledby` will help us simulate the behaviour that a `<legend>` is responsible for, which is linking a title to a group. This difference is important because we have more than one input in a group, and each one of those inputs has their own label, independent of the group title. 
+
+If we are using "prompt" text to provide further explanation about the group, we should use `aria-describedby`, with an id of its own, to provide that description, e.g.:
+```diff
+-<div class="o-forms-field" role="group" aria-labelledby="example-group-title">
++<div class="o-forms-field" role="group" aria-labelledby="example-group-title" aria-describedby="example-group-info">
+	<span class="o-forms-title">
+		<span class="o-forms-title__main" id="example-group-title">Group Title</span>
+-		<span class="o-forms-title__prompt">Group info</span>
++		<span class="o-forms-title__prompt" id="example-group-info">Group info</span>
+		...
+	</span>
+	...
+</div>
+```
+
+`aria-labelledby` and `aria-describedby` are similar attributes, though semantically they provide different functionality.
+You can [read more about them on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute):
+
+>A label provides essential information about an object, while a description provides extended information that the user might need.
+
+#### `aria-label` & `aria-hidden`
+
+Independently of whether or not a group of inputs is in a `<fieldset>` or a `<div>`, their structure exhibits a peculiar type of behaviour: AT will focus on the `<input>`, read out its label, and then focus on the label text, and read that out again. To avoid focusing on the label to provide a more natural 'tabbing' experience, we recommend combining `aria-label` and `aria-hidden` on the grouped inputs:
+
+```diff
+<label>
+-	<input type="checkbox" name="default" value="checkbox1" checked=>
++	<input type="checkbox" name="default" value="checkbox1" checked aria-label="Checkbox 1">
+-	<span class="o-forms-input__label">Checkbox 1</span>
++	<span class="o-forms-input__label" aria-hidden="true">Checkbox 1</span>
+</label>
+```
+
+By defining what the label of the input is and hiding the actual label from the AT, we'll make the transition between inputs far smoother. 
+
+_Note 1: `aria-label` also accepts an id if you would rather set an id on the input's label_
+
+_Note 2: `aria-hidden` has often been used to visually hide/show elements. However, it's primary functionality is to hide the contents of an element from the AT_
```

Comment:
Is this saying we sometimes use `aria-hidden` as a selector to visually hide elements? Took me a while to make the connection. I don't think a user needs to think about that?

Reply:
Many products to exactly that, so I think we do need to point it out

Reply:
Ah okay, maybe a wording tweak?
```suggestion
_Note 2: `aria-hidden` is sometimes used as a CSS selector to visually hide/show elements. However, it's primary functionality is to hide the contents of an element from the AT_
```